snuba/clickhouse/columns.py

```diff
@@ -539,6 +540,12 @@ def get(
         except KeyError:
             return default
 
+    def get_column_names(self) -> Set[str]:
```

Comment:
You don't need this, the class already has a `__contains__` function. You can do `"offset" in <ColumnSet>` directly. Please remove this.